updated emissions documentation (#172)
urbanopt/urbanopt.github.io: workflows/carbon_emissions.md

nav_order: 5
---

# **Carbon Emissions Reporting**

> The following documents the capabilities added to URBANopt&trade; to calculate carbon emissions associated with building electricity, natural gas, propane and fuel oil number 1 use. 

Reviewer: Number 2 or number 1?

    "emissions_annual_historical_year": "2019"
  }
```
# Natural Gas, Propane, and FuelOil #2 Emissions

Emission factors for natural gas, propane, and fuel oil no. 2 are based on EPA eGRID data and calculated using a 20-year global warming potential (GWP) horizon based on ASHRAE 189.1 :
	

> 	natural gas :  277.358126 KG/MWH

Reviewer 2: ```suggestion
	natural gas :  277.358126 kg/MWh
```

is this suppose to be kg/MWh? I'm not sure what KG/MWH is. @rawadelkontar

	propane : 323.896704 KG/MWH

> 	Fueloil #2 : 294.962046 KG/MWH

Reviewer 2: ```suggestion
	Fuel oil #2 : 294.962046 KG/MWH
```

These emission factors are then multiplied by the associated facility total energy use profile for the corresponding fuel type to calculate emissions in metric tons (mt) and emissions intensity in kgCO2e/sqft.

# Outputs

> Emissions results are reported in URBANopt reports. For each feature, timeseries results are reported in the CSV report and aggregate values are reported in URBANopt JSON reports. Bellow are descriptions for the emissions results variables.

Reviewer 2: ```suggestion
Emissions results are reported in URBANopt reports. For each feature, time series results are reported in the CSV report and aggregate values are reported in URBANopt JSON reports. Below are descriptions for the emissions results variables.
```

	Future electricity emissions in metric tons (mt) calculated based on hourly emissions factor values for the selected future year

	"historical_annual_electricity_emissions_mt": 
	Historical electricity emissions in metric tons (mt) calculated based on an annual emissions factor value for the selected historical year

	"historical_hourly_electricity_emissions_mt":  
	Historical electricity emissions in metric tons (mt) calculated based on hourly emissions factor values for the selected historical year

	"future_annual_electricity_emissions_intensity_kg_per_ft2": 
	Future electricity emissions intensity in kg/ft2 calculated based on an annual emissions factor value for the selected future year

	"future_hourly_electricity_emissions_intensity_kg_per_ft2": 
	Future electricity emissions intensity in kg/ft2 calculated based on hourly emissions factor values for the selected future year

	"historical_annual_electricity_emissions_intensity_kg_per_ft2": 
	Historical electricity emissions intensity in kg/ft2 calculated based on an annual emissions factor value for the selected historical year

	"historical_hourly_electricity_emissions_intensity_kg_per_ft2":

> 	Historical electricity emissions intensity in kg.ft2 calculated based on hourly emissions factor values for the selected historical year

Reviewer 2: ```suggestion
	Historical electricity emissions intensity in kg/ft2 calculated based on hourly emissions factor values for the selected historical year
```

	"natural_gas_emissions_mt": 

> 	Natural Gas emissions in mt calculated by multipling the facility natural gas energy use (MWH) by the natural gas emission factor (277.358126 KG/MWH) and convertd to metric ton(mt)

Reviewer 2: ```suggestion
	Natural Gas emissions in metric tonnes (mt) were calculated by multiplying the facility's natural gas energy use (MWh) by the natural gas emission factor (277.4 kg/MWH) and converted to metric tonnes (mt)
```
I think tonnes is the right spelling here? Or maybe not. Another opinion is probably needed.
